packages/terra-i18n-plugin/docs/README.md

```diff
@@ -32,3 +32,12 @@ module.exports = {
 ```
 
 Then all of the messages for a given language across all dependencies will be aggregated into an overall translation file for that language and placed into an aggregated-translations directory.  In order for these resulting files to be able to be loaded by terra-i18n, that directory then needs to be added to the list of directories that the webpack resolver looks into.
+
+By default it will search translation folders in the root node_modules. If an optional option `translationsDirectory` is specified:
```

Comment:
....in the root node_modules. _The i18n-plugin will also search and aggregate translation files under a custom file path when the optional key `translationsDirectory` is specified:_